pkgs/by-name/ps/psqlodbc/package.nix

```diff
@@ -7,27 +7,35 @@ assert lib.xor withLibiodbc withUnixODBC;
 
 stdenv.mkDerivation rec {
   pname = "psqlodbc";
-  version = "16.00.0000";
+  version = "${builtins.replaceStrings ["_"] ["."] (lib.strings.removePrefix "REL-" src.rev)}";
```

Comment:
This looks suspicious to me. I'd like to avoid this if at all possible, and flip the dependency (src depends on version not vice versa.)

Reply:
The version with `rec` (as it is here) is actually less consternation-causing than the version with `finalAttrs`, since it's a "false sharing", only at Nix parse time rather than at derivation parse time. 

Maybe the right call is to make a function somewhere in `pkgs.pkgsLib` or something that accomplishes this, then let `nix-update-script` in on that game.

I don't block the PR on this feedback, btw, just to be clear.